boards/infineon/kit_xmc72_evk/doc/index.rst

```diff
@@ -0,0 +1,71 @@
+.. zephyr:board:: kit_xmc72_evk
+
+Overview
+********
+
+The XMC7200 evaluation kit enables you to evaluate and develop your applications using the XMC7200D microcontroller(hereafter called “XMC7200D”). XMC7200D is designed for industrial applications. XMC7200D is a true programmable embedded system-on chip, integrating up to two 350-MHz Arm® Cortex®-M7 as the primary application processors, 100-MHz Arm® Cortex®-M0+ that supports low-power operations, up to 8-MB flash and 1-MB SRAM, Gigabit Ethernet, Controller Area Network Flexible Data-Rate (CAN FD), Secure Digital Host Controller (SDHC) supporting SD/SDIO/eMMC interfaces, programmable analog and digital peripherals that allow faster time-to-market. The evaluation board has a M.2 interface connector for interfacing radio modules-based on AIROC™ Wi-Fi & Bluetooth combos, SMIF dual header compatible with Digilent Pmod for interfacing HYPERBUS™ memories, and headers compatible with Arduino for interfacing Arduino shields. In addition, the board features an onboard programmer/debugger(KitProg3), a 512-Mbit QSPI NOR flash, CAN FD transceiver, Gigabit Ethernet PHY transceiver with RJ45 connector interface, a micro-B connector for USB device interface, three user LEDs, one potentiometer, and two push buttons. The board supports operating voltages from 3.3 V to 5.0 V for XMC7200D.
+
+Hardware
+********
+
+For more information about XMC7200D and KIT_XMC72_EVK:
+
+- `XMC7200D SoC Website`_
+- `kit_xmc72_evk Board Website`_
+
+Kit Features:
+=============
+
+- Evaluation board for XMC7200D-E272K8384 in BGA package with 272 pins dual-core Arm®Cortex® -M7 CPUs running at - 50-MHz and an Arm® Cortex®-M0+ CPU running at 100-MHz
+- Full-system approach on the board, featuring Gigabit Ethernet PHY and connector, CAN FD transceiver, user LEDs, - uttons, and potentiometer
+- M.2 interface connector for interfacing radio modules based on AIROC™ Wi-Fi & Bluetooth®combos (currently not - supported)
```

Comment:
not sure if you copy-pasted from a PDF or something but you have quite a few typos in there ("[...] user LEDs, - uttons, and [...]" etc)